release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -2809,6 +2809,40 @@ To save the `vmcore` file, use the `crashkernel` setting to reserve 1024 MB of m
 
 * NFS volumes exported from VMware vSAN Files cannot be mounted by clusters running {product-title} 4.19 due to RHEL-83435. To avoid this issue, ensure that you are running VMware ESXi and vSAN at the latest patch versions of 8.0 P05, or later. (link:https://issues.redhat.com/browse/OCPBUGS-55978[OCPBUGS-55978])
 
+Known Gateway API issues
+
+* By default, `istiod` will create a ConfigMap `istio-ca-root-cert` in every namespace it watches, which can create unnecessary ConfigMaps. (link:https://issues.redhat.com/browse/OSSM-9076[OSSM-9076])
+
+* When using Gateway API via the Cluster Ingress Operator (CIO), any Istio specific resources (e.g. VirtualService) are not allowed or supported. (link:https://issues.redhat.com/browse/OSSM-9154[OSSM-9154])
```

Comment:
Not allowed? As In I shouldn't create these; or the server will reject them (which is it)? 
When you say supported; does this mean it breaks something or are we just not testing/validating this (thus its not supported)? 

In short the verbage here leave a lot to the reader to interpret; and we want to where possible remove ambiguity; is there a way we can clean this up to be more explicit about what we mean here?